jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListPresenter.java

```diff
@@ -160,6 +160,13 @@ public void callback(DataSet dataSet) {
         };
     }
 
+    private boolean hasInitiator(String variableName, String variableValue, ProcessInstanceSummary pis) {
```

Comment:
Maybe call it _filterInitiator_ to avoid having to negate the result for better readability.